pages/index.vue

```diff
@@ -1,17 +1,15 @@
 <template>
     <div>
-        <UTable :rows="posts" :columns="columnData" :ui="{ td: { base: 'max-w-[0] truncate' } }">
-            <template #id-data="{ row }">
-                <NuxtLink :to="`post/detail/${row.id}`">{{ row.id }}</NuxtLink>
-            </template>
-            <template #body-data="{ row }">
-                <UTooltip :text="row.body">
-                    {{ row.body }}
-                </UTooltip>
-            </template>
-        </UTable>
-        <div class="flex justify-end px-3 py-3.5 border-t border-gray-200 dark:border-gray-700">
-            <UPagination v-model="currentPage" :page-count="itemsPerPage" :total="totalPosts" :ui="{
+        <UProgress animation="carousel" class="my-4" v-if="isLoaderShow" />
+        <div class="grid grid-cols-3 gap-4">
+            <div v-for="post in posts" :key="post.id" class="p-4 rounded bg-gray-100 cursor-pointer"
+                @click="postDetail(post)">
```

Comment:
please use proper tag, like in this case you should prefer to use anchor tag, it's better for SEO and accessibility as well.

Reply:
But div inside anchor tag is not valid Html for SEO

Reply:
resolved

Reply:
can you please explain it more, I'm curious to know how is it bad for SEO ?

Reply:
if you are using event listener neither browser would be able to identify if it's a link and if we discuss about crowler it would not able to link it as it would not get the link.

Reply:
My bad i just checked, in HTML5 its allow to add div inside anchor